app/models/master_file.rb

```diff
@@ -256,12 +256,21 @@ def process file=nil
       file_dup = file.dup
       file_dup.each_pair {|quality, f| file_dup[quality] = FileLocator.new(f.to_path).uri.to_s }
     else
-      FileLocator.new(file_location).uri.to_s
+      matterhorn_path
     end
 
     ActiveEncodeJob::Create.perform_later(self.id, input, {preset: self.workflow_name})
   end
 
```

Comment:
Avalon has one path for both the file on the app server, and the file on the encoding server. This means that you have to create the same directory structure on both to get a file encoded. The new settings `Rails.application.secrets.matterhorn_client_media_path` and `Rails.application.secrets.matterhorn_server_media_path` allows us to decouple these paths on the two server, and the following `matterhorn_path` method switches the app server path to the matterhorn server path so that the `ActiveEncode` job can find it.